docs/keys.md

```diff
@@ -0,0 +1,62 @@
+# Keys
+
+Document the different Keys in RisingWave.
+
+## Stream Key
+
+The key which can uniquely identify records in the RisingWave stream.
```

Comment:
Seems for a long time we want to introduce a new notion called "unique key" to uniquely identify records. The reason is that stream key can contain part that are unnecessary for uniqueness, like "group key" which is appended to stream key because it's the "distribution key".

Reply:
FYI: https://www.notion.so/risingwave-labs/RisingWave-c81fffbaa18a454488167156f4006fe0
(I'm sorry we only have Chinese references for this topic, I'd like to help with further explanations if you need.

Reply:
> FYI: https://www.notion.so/risingwave-labs/RisingWave-c81fffbaa18a454488167156f4006fe0 (I'm sorry we only have Chinese references for this topic, I'd like to help with further explanations if you need.

Thanks for sharing it